File: Selenium/Activities/Activity11.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC


with webdriver.Firefox() as ff:

    ff.get("https://v1.training-support.net/selenium/dynamic-controls")
    print("title is : ", ff.title)
    wait = WebDriverWait(ff, 10)
    checkbox = ff.find_element(By.XPATH, "//div/input[@type='checkbox']")

    print("Check box is selected :", checkbox.is_selected())

    checkbox.click()
    print("Check box is selected :", checkbox.is_selected())

    ff.quit()

File: Selenium/Activities/Activity11.java
package activities;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity11 {

	public static void main(String[] args) {
		WebDriver driver = new FirefoxDriver();
		driver.get("https://v1.training-support.net/selenium/dynamic-controls");

		System.out.println("The Title of the page is  : " + driver.getTitle());
		WebElement checkbox = driver.findElement(By.xpath("//div/input[@type='checkbox']"));
		
		System.out.println("Check box is selected " + checkbox.isSelected());
		
		checkbox.click();
		
		System.out.println("Check box is selected " + checkbox.isSelected());
		driver.quit();

	}

}
